template/INSTRUCTIONS.md.tpl

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+This document lays out the next steps for AWS service teams upon bootstrapping the ACK {{ .ServicePackageName }}-controller repository successfully.
+
+1. To get started, look inside the `generator.yaml` file, ignore the resource(s) from the resource list and execute `make build-controller` from the ACK code-generator.
+2. Add any custom code inside the `templates` directory, add e2e test inside the `test` directory.
```

Comment:
e2e tests aren't quite as simple as add the tests inside the test directory. Probably better to refer them back to the testing documentation for this, rather than repeating ourselves here

Reply:
Thank you for pointing this out, I have revised the instruction to refer to the testing documentation.